processing/src/main/java/io/druid/query/metadata/SegmentMetadataQueryQueryToolChest.java

```diff
@@ -347,14 +347,23 @@ public static SegmentAnalysis mergeAnalyses(
       mergedId = "merged";
     }
 
+    final Boolean rollup;
+
+    if (arg1.isRollup() != null && arg2.isRollup() != null && arg1.isRollup().equals(arg2.isRollup())) {
+      rollup = arg1.isRollup();
+    } else {
+      rollup = null;
```

Comment:
what does null rollup mean? I feel like whenever we can't make a decision rollup should be set to true with a comment that Druid rolls up data by default